Removed unsupported versions from database products (#5764)
EnterpriseDB/docs: product_docs/docs/epas/13/installing/linux_install_details/managing_an_advanced_server_installation.mdx

> For more information about using the `pg_ctl` utility, or the command line options available, see the official PostgreSQL Core Documentation available at:

Reviewer: ```suggestion
For more information about using the `pg_ctl` utility or the command line options available, see the official [PostgreSQL Core Documentation](https://www.postgresql.org/docs/current/static/app-pg-ctl.html).
```

> [https://www.postgresql.org/docs/current/static/app-pg-ctl.html]<https://www.postgresql.org/docs/current/static/app-pg-ctl.html>

Reviewer: ```suggestion
```

> `edb-psql` is a symbolic link to a binary called `psql`, a modified version of the PostgreSQL community `psql`, with added support for Advanced Server features. . For more information about using the command line client, see the PostgreSQL Core Documentation at:

Reviewer: ```suggestion
`edb-psql` is a symbolic link to a binary called `psql`, a modified version of the PostgreSQL community `psql`, with added support for Advanced Server features.  For more information about using the command line client, see the [PostgreSQL Core Documentation](https://www.postgresql.org/docs/current/static/app-psql.html).
```

> [https://www.postgresql.org/docs/current/static/app-psql.html]<https://www.postgresql.org/docs/current/static/app-psql.html>

Reviewer: ```suggestion
```

The PostgreSQL `initdb` command creates a database cluster; when installing Advanced Server with an RPM package, the `initdb` executable is in `/usr/edb/asx.x/bin`. After installing Advanced Server, you must manually configure the service and invoke `initdb` to create your cluster. When invoking `initdb`, you can:

-   Specify environment options on the command line.
-   Include the `systemd` service manager on RHEL/Rocky Linux/AlmaLinux 8.x and use a service configuration file to configure the environment.

To review the `initdb` documentation, visit:

> [https://www.postgresql.org/docs/current/static/app-initdb.html]<https://www.postgresql.org/docs/current/static/app-initdb.html>

Reviewer: ```suggestion
```

> For more information about using `initdb`, and the available cluster configuration options, see the PostgreSQL Core Documentation available at:

Reviewer: ```suggestion
For more information about using `initdb`, and the available cluster configuration options, see the [PostgreSQL Core Documentation](https://www.postgresql.org/docs/current/static/app-initdb.html).
```

> [https://www.postgresql.org/docs/current/static/app-initdb.html]<https://www.postgresql.org/docs/current/static/app-initdb.html>

Reviewer: ```suggestion
```
